.travis.yml

```diff
@@ -15,15 +15,17 @@ matrix:
       env: TOXENV=py34-django110-alchemy10-mongoengine010
     - python: "3.5"
       env: TOXENV=py35-django110-alchemy10-mongoengine010
+    - python: "3.6"
+      env: TOXENV=py36-django110-alchemy10-mongoengine010
 
     # Pypy
     - python: "pypy"
       env: TOXENV=py27-django110-alchemy10-mongoengine010
 
     # Linting
-    - python: "3.5"
+    - python: "3.6"
```

Comment:
I'd go for replacing: we don't need to run linting/examples on all Python versions ;)